libs/metrics/test_positivity.py

```diff
@@ -126,25 +183,40 @@ def calculate(self, dataset: MultiRegionDataset, diff_days: int) -> MultiRegionD
         # DataFrame without the field label so operators such as `/` are calculated for each
         # region/state and date.
         wide_date_df = delta_df.loc[self._numerator, :] / delta_df.loc[self._denominator, :]
-        return _make_output_dataset(dataset, self._name, wide_date_df, CommonFields.TEST_POSITIVITY)
+
+        return self._filter_recent(
+            _make_output_dataset(dataset, self._name, wide_date_df, CommonFields.TEST_POSITIVITY),
```

Comment:
small nit, but I think it's helpful to have this as its own variable, so i don't have to keep as much context when i'm reading what's happening here

Reply:
done